GeneralsMD/Code/GameEngineDevice/Source/W3DDevice/GameClient/W3DSmudge.cpp

```diff
@@ -467,12 +470,12 @@ void W3DSmudgeManager::render(RenderInfoClass &rinfo)
 	VertexMaterialClass *vmat=VertexMaterialClass::Get_Preset(VertexMaterialClass::PRELIT_DIFFUSE);
 	DX8Wrapper::Set_Material(vmat);
 	REF_PTR_RELEASE(vmat);
-	DX8Wrapper::Apply_Render_State_Changes();
```

Comment:
I removed this call, because `DX8Wrapper::Draw_Triangles` will call this as well and therefore it is redundant.